rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -30,6 +31,19 @@
 
 FREE_NFT_LIMIT = 10
 
+NFT_DB_TUPLE = Tuple[
+    str,  # identifier
+    Optional[str],  # name
+    Optional[str],  # price_in_asset
+    Optional[str],  # price_asset
+    int,  # whether the price is manually input
+    ChecksumEvmAddress,  # owner address
+    bool,  # whether is an lp
```

Comment:
something is odd here. I belive either manually input or is_lp are `bool` or `int` but I wouldn't mix both types